### ISSUE TITLE
IIFE semicolon/newline bug

### ISSUE THREAD

**Issue by feross** [maxogden/standard-format#92]
*Originally posted by @anacrolix, moved from https://github.com/feross/standard/issues/215:*

This kind of code seems quite common:
```js
console.log('meh');
(function a() {
	console.log('hiya');
}());
```

But standard formats it like so:
```js
console.log('meh');(function a () {
  console.log('hiya')
}())
```

**Comment by Flet** [comment]
heh I was just running into this myself this week....

this: 
```js
/*global localStorage*/
(function () { // IIFE to ensure no global leakage!
```

turning into this:
```js
  /*global localStorage*/(function () { // IIFE to ensure no global leakage!
```

**Comment by Flet** [comment]
OK, this should be fixed in the latest published version 1.6.4...

via commit: https://github.com/maxogden/standard-format/commit/bd02eea6de82699730885b09a5f280579efaefc0
...its slightly hacky but it works.

Can someone verify?

**Comment by feross** [comment]
Looks fixed to me!
